docs/requirements.in

```diff
@@ -4,3 +4,14 @@ Sphinx == 4.3.2
 furo >= 2021.11.15
 sphinxcontrib-apidoc >= 0.3.0
 sphinxcontrib-towncrier >= 0.2.0a0
+# See https://github.com/yaml/pyyaml/issues/724#issuecomment-1638587228
+pyyaml==5.3.1
+# PIN to fix:
+# The sphinxcontrib.applehelp extension used by this project needs at least Sphinx v5.0;
+# it therefore cannot be built with this version.
+sphinxcontrib-applehelp==1.0.2
+sphinxcontrib-devhelp==1.0.2
+sphinxcontrib-htmlhelp==2.0.0
+sphinxcontrib-qthelp==1.0.3
+sphinxcontrib-serializinghtml==1.1.5
+sphinxcontrib-towncrier==0.2.1a0
```

Comment:
@webknjaz I am trying to make a release after more than 15-months, but looks like a lot has changed.  I am able to chase down several issues, but this one I am unsure about:

```
python -m tox -e python -- -v
usage: build_isolated.py [global_opts] cmd1 [cmd1_opts] [cmd2 [cmd2_opts] ...]
   or: build_isolated.py --help [cmd1 cmd2 ...]
   or: build_isolated.py --help-commands
   or: build_isolated.py cmd --help

error: option --formats not recognized
ERROR: invocation failed (exit code 1), logfile: /Users/abhinavsingh/Dev/proxy.py/.tox/.package/log/.package-9.log
======================================================================== log start =========================================================================

========================================================================= log end ==========================================================================
ERROR: FAIL could not package project - v = InvocationError("/Users/abhinavsingh/Dev/proxy.py/.tox/.package/bin/python .venv/lib/python3.10/site-packages/tox/helper/build_isolated.py .tox/dist setuptools.build_meta '' ''", 1)
make: *** [lib-pytest] Error 2
```

I am wondering if you have encountered such errors in the past year or so and know any workarounds.   Googling led me to nothing about this error, so I thought of pinging you.

Best
Abhinav

Reply:
Nevermind, sorry for trouble, looks like pinning towncrier helped me get past this issue.

Reply:
Good to know! I'll take a look anyway — perhaps updating the Sphinx extension will address the incompatibility..